curve-ansible/roles/generate_config/templates/client.conf.j2

```diff
@@ -133,6 +133,14 @@ global.logPath={{ client_log_path }}
 # 单元测试情况下
 # logpath=./runlog/
 
+#
+################# read from source conf ###############
+#
+# close fd which read from source file timeout is 300 seconds
+closefd.timeout=300
+# backend thread iterates the fdMap every 600 seconds to close the timeout fd
+closefd.timeInterval=600
```

Comment:
don't write the value of configuration item in here.
refer to other configutation items and write the default value in `curve-ansible/roles/generate_config/defaults/main.yml`.

Reply:
> don't write the value of configuration item in here.
> refer to other configutation items and write the default value in `curve-ansible/roles/generate_config/defaults/main.yml`.

done